examples/get-content-from-multiple-elements/README.md

```diff
@@ -0,0 +1,23 @@
+# Get Content From Multiple Elements
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to extract text content from all items in an `<ul>` list.
+
+The tested page includes a `<ul>` with three `<li>` elements.
+
+During the test the `getTextFromElements` function takes a TestCafe Selector as an argument and returns an array of strings: the `textContent` of each element that matches the `selector`. The test checks if the resulting list matches an expected one with the `t.expect.eql` method.
+
+## TestCafe API Used in This Example
+
+* Test Structure:
+  * [Fixture.page](https://devexpress.github.io/testcafe/documentation/reference/test-api/fixture/page.html) Method
+* Element Identification and Actions:
+  * [Selector](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/) Object
+  * [Selector.find](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/find.html) Method.
+* Assertion and Evaluation:
+  * [t.expect.eql](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/eql.html) Method
+* Custom Scripts:
+  * [ClientFunction](https://devexpress.github.io/testcafe/documentation/reference/test-api/clientfunction/) Object
```

Comment:
I've added a client function reference here since (I assume) this example is supposed to use one.

Reply:
I've refactored the example so that it utilizes `Smart Assertion Query Mechanism` without client functions, so I removed this line and added `Selector.nth`.